assets/base.css

```diff
@@ -3322,7 +3322,7 @@ details-disclosure > details {
   .animate--hover-3d-lift .shopify-challenge__button,
   .animate--hover-3d-lift .customer button,
   .animate--hover-3d-lift .shopify-payment-button__button,
-  .animate--hover-3d-lift .deferred-media__poster-button {
+  .animate--hover-3d-lift .deferred-media .deferred-media__poster-button {
     transition: transform var(--duration-long) ease, box-shadow var(--duration-long) ease;
```

Comment:
Have we considered different approaches for the application of the shadow ?

- Performance wise, wondering if something different would have made sense. [Here](https://tobiasahlin.com/blog/how-to-animate-box-shadow/) they mention using a pseudo element that you hiding/showing by animating the opacity which ends up being less taxing. 
- for focus states we're using `box-shadow` already so that can impact it. For example, when you use tab through the site, the focus states on buttons is now animated and a bit slower/different than non buttons elements
- [there is a bit of poor rendering](https://screenshot.click/27-24-shtt3-cfk5z.mp4) on the quick add button within a card but it's not too bad

Reply:
For product cards, it gets pretty tricky — here are already so many pseudo elements there that it's easy to break things. It might make sense to explore for buttons though. 

The button rendering for the quick add button is a tough one. I see the same thing if I set a 1px border on cards. I think it's just the browser struggling to render such [a thin line at an angle](https://codepen.io/kjellr/pen/MWPodYV) like that. We _might_ be able to improve it using some of the usual rendering hacks (`will-change`, `translateZ`, `backface-visibility`, etc.) but I doubt it — there's only so much you can do when you have a super-thin line like that.